src/appsignal/opentelemetry.py

```diff
@@ -31,6 +31,12 @@ def response_hook(span, request, response):
     DjangoInstrumentor().instrument(response_hook=response_hook)
 
 
+def add_flask_instrumentation():
+    from opentelemetry.instrumentation.flask import FlaskInstrumentor
+
+    FlaskInstrumentor().instrument(enable_commenter=True, commenter_options={})
```

Comment:
Do we want to enable the sql commenter? We didn't for django. Could mess up the grouping of sanitized queries.

Reply:
Indeed, these sorts of comments used to mess up the grouping (Prisma adds them with no option to disable them) so [we changed `sql_lexer` to remove trailing comments](https://github.com/appsignal/sql_lexer/commit/d629f5a8f259e7cfeaf64aa4fb9142e62d59afc3).

So it shouldn't matter, but I agree it's still better to not send a comment just to remove it later.